### ISSUE TITLE
Module currency_symbols.constants not found

### ISSUE THREAD

**Issue by eleazargc** [snoopyjc/xls2xlsx#14]
* xls2xlsx version: 0.1.5
* Python version: 3.8
* Operating System: MacOS

### Description

In module xls2xlsx.htmlxls2xlsx you try to import currency_symbols.constants, however, in currency_symbols last version (2.0.2 lastest as of today) The module constants was renamed to _constants. 

### What I Did

```
try:
    import currency_symbols._constants as currency_symbols_constants
except Exception:
    import importlib
    currency_symbols_constants = importlib.import_module('currency-symbols._constants')
```

**Comment by MksYi** [comment]
Yeah, You are right.  
I also found this question and used the same solve as you.

**Comment by subashhariharan** [comment]
Yes. I am also come here to share the details You shared. Great Job man.

> * xls2xlsx version: 0.1.5
> * Python version: 3.8
> * Operating System: MacOS
> 
> ### Description
> In module xls2xlsx.htmlxls2xlsx you try to import currency_symbols.constants, however, in currency_symbols last version (2.0.2 lastest as of today) The module constants was renamed to _constants.
> 
> ### What I Did
> ```
> try:
>     import currency_symbols._constants as currency_symbols_constants
> except Exception:
>     import importlib
>     currency_symbols_constants = importlib.import_module('currency-symbols._constants')
> ```

**Comment by snoopyjc** [comment]
Fixed in v0.2.0
